features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/mbedtls_device.h

```diff
@@ -20,4 +20,8 @@
 #ifndef MBEDTLS_DEVICE_H
 #define MBEDTLS_DEVICE_H
 
+#ifdef TARGET_UBLOX_EVK_ODIN_W2
+#define MBEDTLS_MPI_WINDOW_SIZE     3 /**< Maximum windows size used. */
+#endif
+
```

Comment:
@RonEld is it ok now?

sorry you were right no need to change MPI_MAX_SIZE. thanks for pointing out